Code/Tools/AzTestRunner/src/main.cpp

```diff
@@ -226,6 +226,12 @@ namespace AzTestRunner
             testMainFunction.reset();
         }
 
+        // Construct a retry command if the test fails
+        if (result != 0)
+        {
+            std::cout << "Retry command: " << std::string(argv[0]) << " " << lib << " " << symbol << std::endl;
```

Comment:
why do you need to construct a string here?
argv[0] should work just fine

Reply:
This line constructs a full retry command that includes the AzTestRunner.exe, the library, and symbol (AzRunUnitTests)

Reply:
what I meant is if there is a reason to not use argv[0] directly:
```suggestion
            std::cout << "Retry command: " << argv[0] << " " << lib << " " << symbol << std::endl;
```

Reply:
argv[0] should just work. std::string(argv[0]) is not a conversion, but a creation of a string (possibly in the heap), which will be less efficient :)

Reply:
You are correct sir. I thought I tried it without the string and it didn't work but I seem to be mistaken. Removing the conversion